test/test_board.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+from constans.constans import (
+    PLAYER_1,
+    PLAYER_2,
+    INITIAL_POSITION_PLAYER_1,
+    INITIAL_POSITION_PLAYER_2
```

Comment:
missing trailing comma at the end of `INITIAL_POSITION_PLAYER_2` and not in alphabetical order
```suggestion
    INITIAL_POSITION_PLAYER_1,
    INITIAL_POSITION_PLAYER_2,
    PLAYER_1,
    PLAYER_2,
```